.github/workflows/push.yml

```diff
@@ -130,7 +130,6 @@ jobs:
         with:
           jitType: tracing
           runTestsParameters: >-
-            -d zend_extension=opcache.so
```

Comment:
On my Windows the `php.exe` has size of 128 KB but the `php_opcache.dll` has size of 463 KB. This is 3.6x binary size increase which might be an issue for embedded devices running php.